TeleSharp.Generator/Program.cs

```diff
@@ -426,7 +432,7 @@ public static string WriteReadCode(Param p)
         public static FileStream MakeFile(string path)
         {
             if (!Directory.Exists(Path.GetDirectoryName(path)))
-                Directory.CreateDirectory(Path.GetDirectoryName(path));
+                Directory.CreateDirectory(Path.GetDirectoryName($"{currentDirectory}../../../../{path}"));
```

Comment:
what is this? this doesn't seem to be related to .NETStandard change, or it's a changed that is not explained in the commit message

Reply:
I've got error with relative path, so it is `cd` to the project root from bin/debug directory

Reply:
what error? this info needs to be in the commit message